styles.css

```diff
@@ -681,3 +697,119 @@ a {
   transform-origin: 1%;
   transition-duration: 0.1s;
 }
+
+/* modal style */
+.modal-container {
+  position: fixed;
+  z-index: 1000;
+  width: 100%;
+  display: none;
+  overflow-y: scroll;
+  justify-content: center;
+  align-items: center;
+  background-color: #c1c7d0;
+  padding-bottom: 50px;
+  padding-top: 50px;
+}
```

Comment:
The popup design is not looking like the one from the template design in the desktop version.

Your design:

<img width="1440" alt="Screen Shot 2022-06-16 at 7 36 13" src="https://user-images.githubusercontent.com/82426682/174071027-6a3d1aa7-5dc3-484f-a4b7-b42936d13203.png">

I shouldn't be able to see the page at the bottom of the popup window. Please make sure you fix its height.

Reply:
Thanks alot @vechicin for the review. for the linters, I have tried fixing it since yesterday.. it passes all checks on my laptop but on github it keeps failing the webhint test. I contacted zendesk and a Snr code reviewer said they haven't found a solution to that problem and I can submit it like that with a note about the problem. If you have a solution, you can also assist me.. For the Height, I will fix it immediately. Thanks alot once again.